packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/ToggleSwitch.tsx

```diff
@@ -65,53 +84,55 @@ export interface ToggleSwitchProps extends Props {
  */
 export const ToggleSwitch = ({
 	checked,
+	cssOverrides,
+	defaultChecked,
 	id,
+	fontWeight = 'regular',
+	fontSize = 'small',
+	format,
 	label,
+	labelBorder = false,
 	labelPosition = 'right',
-	defaultChecked,
-	cssOverrides,
-	onClick = () => undefined,
+	noJs = false,
+	onClick,
 	...props
 }: ToggleSwitchProps): EmotionJSX.Element => {
 	const buttonId = id ?? generateSourceId();
 	const labelId = descriptionId(buttonId);
-	const [isBrowser, setIsBrowser] = useState(false);
-	let tooltiptext = '';
 
 	const isChecked = (): boolean => {
+		if (noJs) return false;
```

Comment:
I'm not sure if it's right to leave this in? It allows us to test but seems slightly odd.